README.md

```diff
@@ -1,556 +1,53 @@
-[![CI](https://github.com/f3d-app/f3d/actions/workflows/ci.yml/badge.svg)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Downloads](https://img.shields.io/reddit/subreddit-subscribers/f3d_app.svg)](https://www.reddit.com/r/f3d_app)
+---
+layout: default
+title: Home
+nav_order: 0
+description: "F3D - Fast and minimalist 3D viewer"
+permalink: /
+---
```

Comment:
This front matter block (and all others like it) appears in the github render of the markdown. There is an issue on github to be able to disable that repository wide but it has not seen much traction:

https://github.com/github/markup/issues/994

Reply:
That's really bad. I'm sure there is a workaround. Any idea?

Reply:
Maybe this: https://github.com/benbalter/jekyll-optional-front-matter

Reply:
Or is it possible to have a `home.md` with the front matter block that includes the content of `readme.md`?

Reply:
the plugin makes front matter optional, but we need it, it contains critical information.

It is possible but it means to duplicate the content, I'd rather not have that.

tbh I dont find it that bad, do you check the visual results ? It comes quite naturally to ignore it:

![a](https://user-images.githubusercontent.com/3129530/200295849-3a2ef6c5-cf10-4b2c-94e8-be4c50d07322.png)

Reply:
I've duplicated README.md into a jekyll specific HOME.md. Lets keep the other file as is.

Reply:
Can you use that: https://jekyllrb.com/docs/includes/#including-files-relative-to-another-file  

In `HOME.md`, you will have something like:

```
{% include_relative README.d %}
```

Reply:
If that works, i will generalize this, Ill let you know

Reply:
just perfect, Ill use that.

Reply:
tunrs out it completely breaks the link logic between doc, but it works for the root doc.